### ISSUE TITLE
[BUG] Only 32 GB mounted on the root directory when the disk size of the repository machine is 64 GB

### ISSUE THREAD

**Issue by przemyslavic** [epiphany-platform/epiphany#1776]
**Describe the bug**
The root partition shows only 32 GB mounted on "/" when the disk size is 64 GB (applies to **Azure/RedHat** configuration).

**To Reproduce**
Deploy a new cluster from develop branch with the configuration given below: `epicli apply -f test.yml`
**Config files**

```yaml
...
  components:
    kafka:
      count: 0
    kubernetes_master:
      count: 0
    kubernetes_node:
      count: 0
    load_balancer:
      count: 0
    logging:
      count: 0
    monitoring:
      count: 0
    postgresql:
      count: 0
    rabbitmq:
      count: 0
    ignite:
      count: 0
    opendistro_for_elasticsearch:
      count: 0
    repository:
      count: 1
      machine:  repository-machine-rhel
---
kind: infrastructure/virtual-machine
name: repository-machine-rhel
provider: azure
based_on: repository-machine
specification:
  storage_image_reference:
    publisher: RedHat
    offer: RHEL
    sku: 7-RAW
    version: 7.7.2019090418
```

**Expected behavior**
There should be ~64GB available mounted on a root directory.

**OS (please complete the following information):**
 - OS: [RHEL7]

**Cloud Environment (please complete the following information):**
 - Cloud Provider [MS Azure]

**Additional context**
```
[operations@ci-devazurrhelcanal-repository-vm-0 ~]$ df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        1.7G     0  1.7G   0% /dev
tmpfs           1.7G     0  1.7G   0% /dev/shm
tmpfs           1.7G  9.1M  1.7G   1% /run
tmpfs           1.7G     0  1.7G   0% /sys/fs/cgroup
/dev/sda2        32G  4.1G   28G  13% /
/dev/sda1       497M  101M  397M  21% /boot
/dev/sdb1       6.8G  2.1G  4.4G  32% /mnt/resource
tmpfs           345M     0  345M   0% /run/user/1000
```
**It looks like if we have a 64 GB disk, we need to additionally resize the partition and the filesystem with `fdisk` and `xfs_growfs`.**

After resizing it is OK.
```
[root@ci-devazurrhelcanal-repository-vm-0 ~]# df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        1.7G     0  1.7G   0% /dev
tmpfs           1.7G     0  1.7G   0% /dev/shm
tmpfs           1.7G  9.1M  1.7G   1% /run
tmpfs           1.7G     0  1.7G   0% /sys/fs/cgroup
/dev/sda2        64G   16G   48G  25% /
/dev/sda1       497M  101M  397M  21% /boot
/dev/sdb1       6.8G   32M  6.4G   1% /mnt/resource
tmpfs           345M     0  345M   0% /run/user/1000
```

**Comment by ar3ndt** [comment]
From configuration perspective it looks all good. For other scenarios (non Azure/RHEL it does not occur).
Hold on until introducing lvm.

**Comment by mkyc** [comment]
This might be solved by #1355. It should be checked again after another one is closed.

**Comment by rpudlowski93** [comment]
Summary: 
- The task can be closed, the main issue of the task is resolved by #1355. 
- We need to increase os disk size from 32GB to 64GB in case of using LVM version of RHEL (7.7, 7.8, 7.9) on Azure. We need to add the information to our documentation (#2080).  
- Additional output of the task is the bug: #2098.
